package.json

```diff
@@ -20,16 +20,46 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:codechecker.helloWorld"
+    "onCommand:codechecker.backend.reloadMetadata",
+    "onView:codechecker.views.overview"
   ],
   "main": "./dist/extension.js",
   "contributes": {
     "commands": [
       {
-        "command": "codechecker.helloWorld",
-        "title": "Hello World"
+        "command": "codechecker.backend.reloadMetadata",
+        "title": "CodeChecker: Reload metadata"
       }
-    ]
+    ],
+    "configuration": {
+      "title": "CodeChecker",
```

Comment:
The build tasks are provided by the Npm extension, which supports various package managers.
I'm not sure this is mentioned anywhere in the official docs, but I'll mention it in the README.

Reply:
Coming back to this, I'm not sure why it failed to execute, since ``npm`` should be a built-in extension.
Is there anything unusual under ``Tasks`` on the Output tab?

Reply:
Added links to the extensions in ``.vscode/extesions,json`` to the README.